src/com/videojs/providers/HTTPVideoProvider.as

```diff
@@ -356,6 +356,7 @@ package com.videojs.providers{
             }
             else if(_hasEnded)
             {
+                _isSeeking = true;
```

Comment:
@imbcmdth pointed out to me that both branches have conditions which may not be mutually exclusive.